### PR TITLE
Add AnimeUnity

### LINKED ISSUE #4363: [Site Request] animeunity
**Name of the website**
animeunity

**Website urls (examples below)**
- Site: https://www.animeunity.tv
**Languages**
- italian
- japanese (ita sub)

**Website relationship*

**Additional details**

### PR DESCRIPTION
**Pull request by MikeZeDev**
Easy anime website : simple API and one damn link by episode, no special video host <3
Fixes https://github.com/manga-download/hakuneko/issues/4363

### CODE REVIEW

File: src/web/mjs/connectors/AnimeUnity.mjs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+export default class AnimeUnity extends Connector {
+    constructor() {
+        super();
+        super.id = 'animeunity';
+        super.label = 'AnimeUnity';
+        this.tags = [ 'anime', 'italian' ];
+        this.url = 'https://www.animeunity.tv';
+        this.token = undefined;
```
Comment (27136773) [inline src/web/mjs/connectors/AnimeUnity.mjs]: If possible avoid fields that hold some kind of temporary state.
Consider to get the token as local variable inside of `_getMangas` and then provide it as parameter for the pagination e.g., `const mangas = this._getMangasFromPage (page, token)`

File: src/web/mjs/connectors/AnimeUnity.mjs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                title: 'Episode '+ element.number,
+            };
+        }).reverse();
+    }
+    async _getPages(chapter) {
+        const uri = new URL(chapter.id, this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'video-player');
+        const episode = JSON.parse(data[0].getAttribute('episode'));
+        return { video : episode.link };
```
Comment (27136773) [inline src/web/mjs/connectors/AnimeUnity.mjs:78]: May include field for subtitle streams in returned object `subtitles: []`

File: src/web/mjs/connectors/AnimeUnity.mjs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    }
+    async getToken() {
+        if (!this.token) {
+            const r = new Request(this.url, this.requestOptions);
```
Comment (27136773) [inline src/web/mjs/connectors/AnimeUnity.mjs]: Use more expressive variable names

**Review by 27136773 — APPROVED** [review]
Minor improvements possible

File: src/web/mjs/connectors/AnimeUnity.mjs (hunk 2 above)
Comment (MikeZeDev) [reply src/web/mjs/connectors/AnimeUnity.mjs:78]: Links are direct to MP4 files, subtitles are hardcoded in videos and not available separately

**Pull request merged**